src/Neo/ProtocolSettings.cs

```diff
@@ -113,8 +113,8 @@ public record ProtocolSettings
             StandbyCommittee = Array.Empty<ECPoint>(),
             ValidatorsCount = 0,
             SeedList = Array.Empty<string>(),
-            MillisecondsPerBlock = 15000,
-            MaxTransactionsPerBlock = 512,
+            MillisecondsPerBlock = 3000,
```

Comment:
I'm not sure we need to change defaults.

Reply:
I think it is better, @roman-khimov , because it may, in the future, reflect in some of our tests that use default. Better to keep consistent.

Reply:
Because we will change testnet only before, it's better to avoid default changes